Katello/katello
# Fixes #28459 - index Pulp 3 package groups and SRPMs

This PR adds support for Pulp 3 package groups and SRPMs.

To test:
1) Create a yum repo with the upstream url https://repos.fedorapeople.org/pulp/pulp/fixtures/srpm/
2) Sync this repo
3) Check that it has 3 SRPMs and 2 package groups (2 errata too).
